src/components/calcite-pick-list/calcite-pick-list.scss

```diff
@@ -21,7 +21,7 @@
     justify-end
     items-stretch
     mb-1;
-  &.sticky {
+  &.sticky-pos {
```

Comment:
https://github.com/tailwindlabs/tailwindcss/issues/6386#issuecomment-997077508

Reply:
Looks like we need to make sure our classes never match tailwind utils. We could look into creating a stylelint rule or similar to prevent circular dependencies.